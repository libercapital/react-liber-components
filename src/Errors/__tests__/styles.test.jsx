import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import {
  DesktopContainer,
  MobileContainer,
  MobileBackground,
} from '../styles';

describe('Testing Errors styles', () => {
  configure({ adapter: new Adapter() });
  it('DesktopContainer Should render correctly', () => {
    const wrapper = shallow(<DesktopContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('MobileContainer Should render correctly', () => {
    const wrapper = shallow(<MobileContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('MobileBackground Should render correctly', () => {
    const wrapper = shallow(<MobileBackground />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
