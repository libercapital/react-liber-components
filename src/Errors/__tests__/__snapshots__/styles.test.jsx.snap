// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Errors styles DesktopContainer Should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__DesktopContainer-sc-1f8nl4c-1",
        "isStatic": false,
        "rules": Array [
          "color:#2E2F30;text-align:center;line-height:2em;font-family:Roboto,sans-serif;margin:0;position:relative;width:100vw;height:100vh;overflow:hidden;",
          "display:none;",
          "@media (min-width:",
          "992px",
          "){",
          "
    display: block;
  ",
          "}",
        ],
      },
      "displayName": "styles__DesktopContainer",
      "foldedComponentIds": Array [
        "styles__Container-sc-1f8nl4c-0",
      ],
      "render": [Function],
      "styledComponentId": "styles__DesktopContainer-sc-1f8nl4c-1",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Testing Errors styles MobileBackground Should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__MobileBackground-sc-1f8nl4c-3",
        "isStatic": false,
        "rules": Array [
          "position:absolute;top:0;left:0;width:100vw;height:100vh;z-index:-1;",
        ],
      },
      "displayName": "styles__MobileBackground",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "styles__MobileBackground-sc-1f8nl4c-3",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Testing Errors styles MobileContainer Should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "styles__MobileContainer-sc-1f8nl4c-2",
        "isStatic": false,
        "rules": Array [
          "color:#2E2F30;text-align:center;line-height:2em;font-family:Roboto,sans-serif;margin:0;position:relative;width:100vw;height:100vh;overflow:hidden;",
          "display:block;",
          "@media (min-width:",
          "992px",
          "){",
          "
    display: none;
  ",
          "}",
        ],
      },
      "displayName": "styles__MobileContainer",
      "foldedComponentIds": Array [
        "styles__Container-sc-1f8nl4c-0",
      ],
      "render": [Function],
      "styledComponentId": "styles__MobileContainer-sc-1f8nl4c-2",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
